src/homework/day8/collections/MyCities.java

```diff
@@ -0,0 +1,33 @@
+package homework.day8.collections;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyCities {
+
+    public static void main(String[] args) {
+
```

Comment:
Extra new line. Remove. The same for such cases.

Reply:
Extra line is removed.